docs/tempo/website/metrics-generator/app-performance-mgmt.md

```diff
@@ -0,0 +1,167 @@
+
+## Use filters to reveal details
+
+The APM dashboard provides a gateway to your tracing information in the service graphs and span metrics. 
+his dashboard is derived from a fixed set of metrics queries.
+These underlying queries can not be changed.
+However, you can choose which traces are included in the metrics query by filtering.
+
+You can explore data by clicking on selectable items or by using filters.
+
+### Selecting items or nodes for more detail
+
+Clicking on selectable items, such as underlined text in the APM table or nodes on the service graph, lets you reveal specific details based upon your selection.
+
+In the APM table, you can select items in the **Rate**, **Error Rate**, **Duration (p90)**, and **Links** columns. Choosing one if these items provides details about the span metrics.
+
+<p align="center"><img src="../apm-rate-drilldown.png" alt="APM table with rate drill-down"></p>
+
+
+You can view request rate, request histogram, failed request rate, and traces for any node in the service graph.
+To view more information, select the node in the service graph and then choose an option from the popup.
+For details on navigating the service graph, refer to the [Node graph panel](https://grafana.com/docs/grafana/next/visualizations/node-graph/) documentation.
+
+<p align="center"><img src="../apm-service-graph-drilldown.png" alt="APM service graph with drill-down"></p>
+
+
+### Filter with metric queries
+
+Using the filters at the top of the screen, you can narrow the data set based upon span attributes (key-value pairs or labels).
+The filters build a query to refine what is shown in the service graph and span metrics.
+You can add one or more label filters.
+
+To use the filters:
+
+1. At the top of the Service Graph, select the text box after **Filter** to display a list of available labels. In this case, **server** is selected. <br /><img src="../apm-query-filter-label.png" alt="APM service graph with grid layout">
+
+1. Select or search for a value for the label. In this case, the value of **server** is equal to **tempo-ingester**. The default operator is equals (=). <br /> <img src="../apm-query-filter-value1.png" alt="APM select value for label">
+
+1. Optional: Change the operator by selecting **=** and choosing a new option from the drop-down. <br /><img src="../apm-filter-operator.png" alt="APM filter operators">
+
+
+1. Optional: Add additional key-value pairs to refine the data set. Any subsequent label filters use AND, which requires both key-value pairs to be presents for matches.
+
+1. Select **Run query**.
+
+Filters can be removed by selecting the filter drop-down and choosing **– remove filter –**.
+
+<p align="center"><img src="../apm-remove-filter.png" alt="APM remove filters"></p>
+
+In the example below, each field or label represents a key-value pair. Number 1 selects a service as the label whose value is Go-http-client (2). The second key-value pair has a client as a label whose value is 02e807.
```

Comment:
```suggestion
In the example below, each field or label represents a key-value pair. Number 1 selects a service as the label whose value is `Go-http-client` (2). The second key-value pair has a client as a label whose value is `02e807`.
```

Suggestion to make more readable. Idk if this conforms to our style guide though :)

Reply:
Nope, that works fine. And follows how I've been trying to do the other examples as well.